README.md

```diff
@@ -1,78 +1,157 @@
 
-You can [ learn more here>>](https://stylelint.io/user-guide/usage/cli/#autofixing-errors)
+To get a local copy up and running follow these simple steps.
 
-## Live Demo
+### Prerequisites
 
-[Live Demo Link](https://rozhi-98.github.io/awesome-books/)
+- Use `git clone git@github.com:rozhi-98/Awesome-books-ES6-modules.git` to clone the repository using the SSH link into your local machine
 
-## Author
+### Setup
+
+- `cd` into your local repository
+
+### Install
+
+- `npm install` - Installs the necessary dependencies
+
+### Usage
+
+- Open the index file and enjoy
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
 
 👤 **Rozh Bahman Tahir**
 
-- GitHub: https://github.com/rozhi-98
-- LinkedIn: https://www.linkedin.com/in/rozh-tahir-446203185
+- GitHub: [@githubhandle](https://github.com/rozhi-98)
+- Twitter: [@twitterhandle](https://twitter.com/rozhi98?t=xS_0o-oG7K5cgodULjesNA&s=09)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/rozh-bahman-tahir-446203185)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Add unique ID for each book]**
+- [ ] **[Add date and time with Luxon library]**
+- [ ] **[Create contact page]**
 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-## 🤝 Contributing
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](../../issues/).
+Feel free to check the [issues page](https://github.com/rozhi-98/Awesome-books-ES6-modules/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+If you like this project please support us by giving it a ⭐️.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Cindy Shin in Behance for the original design idea: [Cindy Shin in Behance](https://www.behance.net/adagio07).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ <a name="faq"></a>
+
+- **[What frameworks have you used?]**
 
-## Show your support
+  - [We have not used any frameworks]
 
-Give a ⭐️ if you like this project!
+- **[Why did you work in group for a simple project?]**
 
-## Acknowledgments
+  - [Working in group is a habit of real work places and is good practice.]
 
-- Original design idea by [Cindy Shin in Behance](https://www.behance.net/adagio07).
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-## 📝 License
+<!-- LICENSE -->
 
-This project is [MIT](./LICENSE.txt) licensed.
+## 📝 License <a name="license"></a>
 
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] OPTIONAL
MIT license link is broken in your ReadMe, kindly endeavor to fix it.